python/ray/rllib/agents/a3c/a3c_tf_policy_graph.py

```diff
@@ -155,6 +155,12 @@ def extra_compute_grad_fetches(self):
     def extra_compute_action_fetches(self):
         return {"vf_preds": self.vf}
 
+    @override(TFPolicyGraph)
```

Comment:
Is it possible to auto generate these based on the other method defs for extra fetches? I'm worried this will be hard to keep in sync for all algorithms.